src/doom/d_pwad.c

```diff
@@ -443,6 +445,30 @@ void D_LoadNerveWad (void)
 	}
 }
 
+// [crispy] check if the single MASTERLEVELS.WAD is the kex 2024 version
+static void CheckMasterlevelKex (void)
+{
+	int		lumpnum;
+	patch_t *patch;
+
+	if (masterlevels_kex)
+	{
+		return;
+	}
+	// kex instead of psn/unity MASTERLEVELS.WAD?
+	lumpnum = W_CheckNumForName("MWILV19");
+	if (lumpnum == -1)
+	{
+		// loaded as PWAD
+		lumpnum = W_CheckNumForName("CWILV19");
+	}
+	patch = W_CacheLumpNum(lumpnum, PU_STATIC);
+	if (patch != NULL && patch->width == 258)
```

Comment:
What is special about this patch and why is it expected to have this width? This needs a comment. Actually, I'd prefer if you could compare the widths of two patches of which one is wider than the other in the kex release and vice versa in the psn release.